sdk/src/connections/network/network.cpp

```diff
@@ -386,7 +386,8 @@ int Network::callback_function(struct lws *wsi,
             }
 
             // process message
-            google::protobuf::io::ArrayInputStream ais(in, len);
+            google::protobuf::io::ArrayInputStream ais(
+                in, static_cast<unsigned int>(len));
```

Comment:
shouldn't be 'int' instead of 'unsigned int' ?

Reply:
Yes, it's int.